behaviours/sam_diving_controller/sam_diving_controller/ParamUtils.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+
+import rclpy
+import sys
+
+class DivingModelParam():
+
+    def __init__(self, node):
+
+        self._node = node
+
+        self._node.declare_parameter('vbs_pid_kp', 40.0) 
+        self._node.declare_parameter('vbs_pid_ki', 5.0)
+        self._node.declare_parameter('vbs_pid_kd', 1.0)
+        self._node.declare_parameter('vbs_pid_kaw', 1.0)
+        self._node.declare_parameter('vbs_u_neutral', 50.0)
+        self._node.declare_parameter('vbs_u_min', 0.0)
+        self._node.declare_parameter('vbs_u_max', 100.0)
+        self._node.declare_parameter('vbs_u_emergency', 0.0)
+
+        self._node.declare_parameter('lcg_pid_kp', 40.0)
+        self._node.declare_parameter('lcg_pid_ki', 5.0)
+        self._node.declare_parameter('lcg_pid_kd', 1.0)
+        self._node.declare_parameter('lcg_pid_kaw', 1.0)
+        self._node.declare_parameter('lcg_u_neutral', 50.0)
+        self._node.declare_parameter('lcg_u_min', 0.0)
+        self._node.declare_parameter('lcg_u_max', 100.0)
+        self._node.declare_parameter('lcg_u_emergency', 50.0)
+
+        self._node.declare_parameter('tv_pid_kp', 2.5)
+        self._node.declare_parameter('tv_pid_ki', 0.25)
+        self._node.declare_parameter('tv_pid_kd', 0.5)
+        self._node.declare_parameter('tv_pid_kaw', 1.0)
+        self._node.declare_parameter('tv_u_neutral', 0.0)
+        self._node.declare_parameter('tv_u_min', -0.122173)
+        self._node.declare_parameter('tv_u_max', 0.122173)
+        self._node.declare_parameter('tv_u_emergency', 0.0)
+
+        self._node.declare_parameter('yaw_pid_kp', 2.5)
+        self._node.declare_parameter('yaw_pid_ki', 0.25)
+        self._node.declare_parameter('yaw_pid_kd', 0.5)
+        self._node.declare_parameter('yaw_pid_kaw', 1.0)
+        self._node.declare_parameter('yaw_u_neutral', 0.0)
+        self._node.declare_parameter('yaw_u_min', -0.122173)
+        self._node.declare_parameter('yaw_u_max', 0.122173)
+
+        self._node.declare_parameter('rpm_u_neutral', 0.0)
+        self._node.declare_parameter('rpm_u_min', -400.0)
+        self._node.declare_parameter('rpm_u_max', 1500.0)
+        self._node.declare_parameter('rpm_u_emergency', 0.0)
+
+        self._node.declare_parameter('max_dive_pitch', 0.349)
+
+    def get_param(self):
```

Comment:
You can simplify this using the `__getitem__` method. Example: https://github.com/smarc-project/smarc2/blob/c03d962b23e538bda33ec02b3e2174f66768145e/behaviours/smarc_bt/smarc_bt/vehicles/sensor.py#L65
`def __getitem__(self, key): return self._node.get_parameter(key).get_parameter_value().double_value`

Then you can do `DivingModelParam['vbs_pid_kp']` to replace `param = DivingModelParam.param; param['vbs_pid_kp']` etc.